capstone-app/src/app/modules/photos/components/image-container/image-container.component.ts

```diff
@@ -0,0 +1,56 @@
+import { MockPhotoFetcher } from './../../services/mock-photo-fetcher.service';
```

Comment:
These imports are probably too many levels back to use relative path. I'd recommend using an absolute path if you need more than one ".."